DQM/Integration/python/config/fileinputsource_cfi.py

```diff
@@ -78,6 +78,7 @@
 runstr = str(options.runNumber)
```

Comment:
are you sure this changes belong to your developments for trigger DQM/validation ?